package exceptionDemo;

import java.io.IOException;
import java.nio.file.FileAlreadyExistsException;

public class parentexception {
	
	public void check()throws IOException
	{ 
		System.out.println("c - p");
	}
}
